services/api/mw2pdf/src/utils/cookies.ts

```diff
@@ -1,13 +1,37 @@
+import { Response } from 'node-fetch';
 
-export function parseCookies(domain, response) {
+/**
+ * An interface that describes the structure of a
+ * parsed cookie.
+ */
+export interface ParsedCookie {
+  name: string;
+  value: string;
+  domain: string; 
+}
+
+/**
+ *
+ * Parse cookies for a particular domain from a fetch response.
+ *
+ * @param domain The domain of the cookies.
+ * @param response A [[Response]] from which to harvest cookies of the `domain` domain.
+ */
+export function parseCookies(domain: string, response: Response): Array<ParsedCookie>  {
   const raw = response.headers.raw()['set-cookie'];
   return raw.map((entry) => {
     const [cookie] = entry.split(';')
     const [name, value] = cookie.split('=')
-    return { name, value, domain};
+    return { name, value, domain };
   });
 }
 
-export function toCookieString(cookies) {
-  return cookies.map(cookie => `${cookie.name}=${cookie.value}`).join(';')
+/**
+ *
+ * Serialize array of parsed cookies into a string.
+ *
+ * @param cookies An array of [[ParsedCookie]]s from which to create a string.
+ */
+export function toCookieString(cookies: Array<ParsedCookie>): string {
+  return cookies.map(cookie => `${cookie.name}=${cookie.value}`).join(';');
```

Comment:
lurking semicolon change here

Reply:
I took all the semis out of this file! I think that was the Right Thing (TM) to do.